getyourdata/organization/tests.py

```diff
@@ -506,181 +544,85 @@ def setUp(self):
 
             organization.authentication_fields.add(self.auth_field1)
 
-    def test_cant_create_request_with_no_selected_organizations(self):
         self.selenium.get(
             "%s%s" % (self.live_server_url,
                       reverse("organization:list_organizations")))
 
+    def test_cant_create_request_with_no_selected_organizations(self):
```

Comment:
In this test we start off on any possible page that was left after the last test, which makes this test easy to break by accident.